scss/components/_buttons.scss

```diff
@@ -44,7 +44,7 @@ $DBbutton-border-tertiary: color(blue) !default;
 
     &[disabled], &#{&}--disabled {
         opacity: 0.5;
-
+        pointer-events: none;
```

Comment:
Properties should be ordered opacity, cursor, pointer-events<br>Property pointer-events should be separated from the previous group of properties ending with opacity